### PR TITLE
fix: Delay iOS focus request after load

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue: https://github.com/unoplatform/tradezero-private/issues/43

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs [hunk 1]
```diff
@@ -145,7 +145,17 @@ private static void IsAutoFocusChanged(DependencyObject d, DependencyPropertyCha
 		private static void OnLoaded(object sender, RoutedEventArgs e)
 		{
 			var control = sender as Control;
-			ApplyAutoFocus(control);
+#if !__IOS__ && !__MACOS__
+			ApplyAutoFocus(control);			
+#else
+			// When UITextField is focused right after it is moved to window,
+			// the first responder request will succeed, but the keyboard will
+			// not be displayed - we must delay the request slightly.
```
Comment (jeromelaban) [inline src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs:157]: Let's add a reference to the uno issue as well, thanks!

Comment (MartinZikmund) [reply src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs:157]: Done 👍

**Review by jeromelaban — APPROVED** [review]

**Review by jeanplevesque — APPROVED** [review]

File: src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs [hunk 2]
```diff
@@ -145,7 +145,18 @@ private static void IsAutoFocusChanged(DependencyObject d, DependencyPropertyCha
...
+			// https://github.com/unoplatform/uno/issues/9003
+			await control.Dispatcher.RunIdleAsync(d =>
```
Comment (jeanplevesque) [inline src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs:159]: This code doesn't build.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**